MeesGame/Gameplay/GameManagement/GameEnvironment.cs

```diff
@@ -5,11 +5,13 @@
 
 public class GameEnvironment : Game
 {
+    public static event EventHandler ScreenChanged;
+
     protected GraphicsDeviceManager graphics;
     protected SpriteBatch spriteBatch;
     protected InputHelper inputHelper;
     protected Matrix spriteScale;
-    protected Point windowSize;
+    protected Point preferedWindowSize;
```

Comment:
I've renamed WindowSize to PreferedWindowSize because the window size isn't necessarily equal to this variable.